pkg/clients/github/pull_request.go

```diff
@@ -84,3 +87,52 @@ func (g *Github) GetPRDetails(ghRepo string, prID int) (string, string, error) {
 	}
 	return *pullRequest.Head.Repo.CloneURL, *pullRequest.Head.Ref, nil
 }
+
+// GetCheckRunConclusion fetches a specific CheckRun within a given repo
+// by matching the CheckRun's name with the given checkRunName, and
+// then returns the CheckRun conclusion
+func (g *Github) GetCheckRunConclusion(checkRunName, repoName, prHeadSha string, prNumber int) (string, error) {
+	const checkrunStatusCompleted = "completed"
```

Comment:
sorry but wdyt about moving this to `pkg/constants/constants.go` file?

Reply:
yeah sure thing